CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Is this just a copy of our existing one or did you adapt anything?

Reply:
The default code of conduct, just in the newest version